package.yaml

```diff
@@ -33,6 +33,7 @@ dependencies:
 - blaze-html == 0.9.*
 - bytestring == 0.11.*
 - containers == 0.6.*
+- deepseq == 1.4.*
```

Comment:
I don't think we really need this dependency. See comments in the evaluator file.

Reply:
Done.

Reply:
Done.